content/en/containers/faq/build_cluster_agent.md

```diff
@@ -1,8 +1,9 @@
 ---
 title: Build the Datadog Cluster Agent
-kind: documentation
+kind: faq
```

Comment:
Always surprised to see things move into the FAQ, so just want to check that it's very intentional

Reply:
this one's intentional—info doesn't seem super useful to most people but i don't want to delete it (yet)